DuckDuckGo/AppDelegate.swift

```diff
@@ -410,11 +413,39 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             WindowsBrowserWaitlist.shared.sendInviteCodeAvailableNotification()
         }
 
+        VPNWaitlist.shared.fetchInviteCodeIfAvailable { [weak self] error in
```

Comment:
Maybe this isn’t the time, but at some point we should probably think of stopping adding so much code directly to the `AppDelegate`

Reply:
This doesn't solve the problem of the app delegate doing too much, but I've moved the waitlist functionality out into an AppDelegate extension.